+ source /root/mugen-riscv/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Wed Dec  7 09:03:50 2022 - INFO  - Start to prepare the test environment.
+ DNF_INSTALL 'lxc lxc-devel lxc-libs lxcfs lxcfs-tools tar busybox'
+ pkgs='lxc lxc-devel lxc-libs lxcfs lxcfs-tools tar busybox'
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs 'lxc lxc-devel lxc-libs lxcfs lxcfs-tools tar busybox' --node 1 --tempfile ''
Wed Dec  7 09:03:58 2022 - ERROR - pkgs:(lxcfs-tools busybox) not found
+ tmpfile2=
+ '[' -z '' ']'
+ tmpfile=
++ rpm -qa lxc
++ awk -F - '{print $2}'
+ version=
+ LOG_INFO 'End to prepare the test environment.'
+ message='End to prepare the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment.'
Wed Dec  7 09:04:00 2022 - INFO  - End to prepare the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Wed Dec  7 09:04:00 2022 - INFO  - Start to run test.
+ lxc-create -t /usr/share/lxc/templates/lxc-busybox -n myEuler1
oe_test_lxc_info.sh: line 30: lxc-create: command not found
+ CHECK_RESULT 127 0 0 'Failed to set up container.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Failed to set up container.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Failed to set up container.'
+ LOG_ERROR 'Failed to set up container.'
+ message='Failed to set up container.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed to set up container.'
Wed Dec  7 09:04:01 2022 - ERROR - Failed to set up container.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lxc_info.sh line 31'
+ message='oe_test_lxc_info.sh line 31'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lxc_info.sh line 31'
Wed Dec  7 09:04:02 2022 - ERROR - oe_test_lxc_info.sh line 31
+ return 0
+ lxc-start myEuler1
oe_test_lxc_info.sh: line 32: lxc-start: command not found
+ CHECK_RESULT 127 0 0 'Failed to start container.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Failed to start container.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Failed to start container.'
+ LOG_ERROR 'Failed to start container.'
+ message='Failed to start container.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Failed to start container.'
Wed Dec  7 09:04:02 2022 - ERROR - Failed to start container.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lxc_info.sh line 33'
+ message='oe_test_lxc_info.sh line 33'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lxc_info.sh line 33'
Wed Dec  7 09:04:03 2022 - ERROR - oe_test_lxc_info.sh line 33
+ return 0
+ lxc-info --help
+ grep -i Usage
+ CHECK_RESULT 1 0 0 'Check lxc-info --help failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check lxc-info --help failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check lxc-info --help failed.'
+ LOG_ERROR 'Check lxc-info --help failed.'
+ message='Check lxc-info --help failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check lxc-info --help failed.'
Wed Dec  7 09:04:03 2022 - ERROR - Check lxc-info --help failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lxc_info.sh line 36'
+ message='oe_test_lxc_info.sh line 36'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lxc_info.sh line 36'
Wed Dec  7 09:04:04 2022 - ERROR - oe_test_lxc_info.sh line 36
+ return 0
+ lxc-info --usage
+ grep -i 'Usage: lxc-info'
+ CHECK_RESULT 1 0 0 'Check lxc-info --usage failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check lxc-info --usage failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check lxc-info --usage failed.'
+ LOG_ERROR 'Check lxc-info --usage failed.'
+ message='Check lxc-info --usage failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check lxc-info --usage failed.'
Wed Dec  7 09:04:05 2022 - ERROR - Check lxc-info --usage failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lxc_info.sh line 38'
+ message='oe_test_lxc_info.sh line 38'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lxc_info.sh line 38'
Wed Dec  7 09:04:05 2022 - ERROR - oe_test_lxc_info.sh line 38
+ return 0
+ lxc-info --version
+ grep
oe_test_lxc_info.sh: line 39: lxc-info: command not found
Usage: grep [OPTION]... PATTERNS [FILE]...
Try 'grep --help' for more information.
+ CHECK_RESULT 2 0 0 'Check lxc-info --version failed.'
+ actual_result=2
+ expect_result=0
+ mode=0
+ error_log='Check lxc-info --version failed.'
+ '[' -z 2 ']'
+ '[' 0 -eq 0 ']'
+ test 2x '!=' 0x
+ test -n 'Check lxc-info --version failed.'
+ LOG_ERROR 'Check lxc-info --version failed.'
+ message='Check lxc-info --version failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check lxc-info --version failed.'
Wed Dec  7 09:04:06 2022 - ERROR - Check lxc-info --version failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lxc_info.sh line 40'
+ message='oe_test_lxc_info.sh line 40'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lxc_info.sh line 40'
Wed Dec  7 09:04:06 2022 - ERROR - oe_test_lxc_info.sh line 40
+ return 0
+ lxc-info myEuler1
+ grep -i State
oe_test_lxc_info.sh: line 42: lxc-info: command not found
+ grep -i RUNNING
+ CHECK_RESULT 1 0 0 'Check lxc-info failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check lxc-info failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check lxc-info failed.'
+ LOG_ERROR 'Check lxc-info failed.'
+ message='Check lxc-info failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check lxc-info failed.'
Wed Dec  7 09:04:07 2022 - ERROR - Check lxc-info failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lxc_info.sh line 43'
+ message='oe_test_lxc_info.sh line 43'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lxc_info.sh line 43'
Wed Dec  7 09:04:08 2022 - ERROR - oe_test_lxc_info.sh line 43
+ return 0
+ lxc-info -n myEuler1
+ grep -i seconds
+ grep -i 'CPU use:'
oe_test_lxc_info.sh: line 44: lxc-info: command not found
+ CHECK_RESULT 1 0 0 'Check lxc-info -n failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check lxc-info -n failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check lxc-info -n failed.'
+ LOG_ERROR 'Check lxc-info -n failed.'
+ message='Check lxc-info -n failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check lxc-info -n failed.'
Wed Dec  7 09:04:08 2022 - ERROR - Check lxc-info -n failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lxc_info.sh line 45'
+ message='oe_test_lxc_info.sh line 45'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lxc_info.sh line 45'
Wed Dec  7 09:04:09 2022 - ERROR - oe_test_lxc_info.sh line 45
+ return 0
+ lxc-info -p myEuler1
+ grep -i PID:
oe_test_lxc_info.sh: line 46: lxc-info: command not found
+ CHECK_RESULT 1 0 0 'Check lxc-info -p failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check lxc-info -p failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check lxc-info -p failed.'
+ LOG_ERROR 'Check lxc-info -p failed.'
+ message='Check lxc-info -p failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check lxc-info -p failed.'
Wed Dec  7 09:04:09 2022 - ERROR - Check lxc-info -p failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lxc_info.sh line 47'
+ message='oe_test_lxc_info.sh line 47'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lxc_info.sh line 47'
Wed Dec  7 09:04:10 2022 - ERROR - oe_test_lxc_info.sh line 47
+ return 0
+ lxc-info -S myEuler1
+ grep -i 'Memory use:'
oe_test_lxc_info.sh: line 48: lxc-info: command not found
+ CHECK_RESULT 1 0 0 'Check lxc-info -S failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check lxc-info -S failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check lxc-info -S failed.'
+ LOG_ERROR 'Check lxc-info -S failed.'
+ message='Check lxc-info -S failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check lxc-info -S failed.'
Wed Dec  7 09:04:11 2022 - ERROR - Check lxc-info -S failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lxc_info.sh line 49'
+ message='oe_test_lxc_info.sh line 49'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lxc_info.sh line 49'
Wed Dec  7 09:04:11 2022 - ERROR - oe_test_lxc_info.sh line 49
+ return 0
+ lxc-info -s myEuler1
+ grep -i RUNNING
oe_test_lxc_info.sh: line 50: lxc-info: command not found
+ CHECK_RESULT 1 0 0 'Check lxc-info -s failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check lxc-info -s failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check lxc-info -s failed.'
+ LOG_ERROR 'Check lxc-info -s failed.'
+ message='Check lxc-info -s failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check lxc-info -s failed.'
Wed Dec  7 09:04:12 2022 - ERROR - Check lxc-info -s failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lxc_info.sh line 51'
+ message='oe_test_lxc_info.sh line 51'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lxc_info.sh line 51'
Wed Dec  7 09:04:13 2022 - ERROR - oe_test_lxc_info.sh line 51
+ return 0
+ lxc-info -H myEuler1
+ grep -i 'BlkIO use:'
oe_test_lxc_info.sh: line 52: lxc-info: command not found
+ CHECK_RESULT 1 0 0 'Check lxc-info -H failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check lxc-info -H failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check lxc-info -H failed.'
+ LOG_ERROR 'Check lxc-info -H failed.'
+ message='Check lxc-info -H failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check lxc-info -H failed.'
Wed Dec  7 09:04:13 2022 - ERROR - Check lxc-info -H failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lxc_info.sh line 53'
+ message='oe_test_lxc_info.sh line 53'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lxc_info.sh line 53'
Wed Dec  7 09:04:14 2022 - ERROR - oe_test_lxc_info.sh line 53
+ return 0
+ lxc-stop myEuler1
oe_test_lxc_info.sh: line 55: lxc-stop: command not found
+ lxc-info -s myEuler1
+ grep -i STOPPED
oe_test_lxc_info.sh: line 56: lxc-info: command not found
+ CHECK_RESULT 1 0 0 'Check lxc-info -s failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check lxc-info -s failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check lxc-info -s failed.'
+ LOG_ERROR 'Check lxc-info -s failed.'
+ message='Check lxc-info -s failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check lxc-info -s failed.'
Wed Dec  7 09:04:14 2022 - ERROR - Check lxc-info -s failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lxc_info.sh line 57'
+ message='oe_test_lxc_info.sh line 57'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lxc_info.sh line 57'
Wed Dec  7 09:04:15 2022 - ERROR - oe_test_lxc_info.sh line 57
+ return 0
+ lxc-info -n myEuler1
+ grep -i State
oe_test_lxc_info.sh: line 58: lxc-info: command not found
+ grep -i STOPPED
+ CHECK_RESULT 1 0 0 'Check lxc-info -n failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check lxc-info -n failed.'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check lxc-info -n failed.'
+ LOG_ERROR 'Check lxc-info -n failed.'
+ message='Check lxc-info -n failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check lxc-info -n failed.'
Wed Dec  7 09:04:16 2022 - ERROR - Check lxc-info -n failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_lxc_info.sh line 59'
+ message='oe_test_lxc_info.sh line 59'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_lxc_info.sh line 59'
Wed Dec  7 09:04:16 2022 - ERROR - oe_test_lxc_info.sh line 59
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Wed Dec  7 09:04:17 2022 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 13
+ test 13 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Wed Dec  7 09:04:17 2022 - ERROR - failed to execute the case.
+ exit 13
+ post_test
+ LOG_INFO 'Start to restore the tet environment.'
+ message='Start to restore the tet environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to restore the tet environment.'
Wed Dec  7 09:04:18 2022 - INFO  - Start to restore the tet environment.
+ lxc-destroy myEuler1
oe_test_lxc_info.sh: line 65: lxc-destroy: command not found
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z '' ]]
+ [[ -z '' ]]
+ LOG_WARN 'no thing to do.'
+ message='no thing to do.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level warn --message 'no thing to do.'
Wed Dec  7 09:04:18 2022 - WARN  - no thing to do.
+ return 0
+ LOG_INFO 'End to restore the tet environment.'
+ message='End to restore the tet environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to restore the tet environment.'
Wed Dec  7 09:04:19 2022 - INFO  - End to restore the tet environment.
